### PR TITLE
Fullstack demo (simple command-line example)

### PR DESCRIPTION
**Pull request by subfuzion**
Simple command-line example (`frontend-cli`) to augment the Flutter app (`frontend`) for connecting to the `backend`.

### CODE REVIEW

**Review by grant — APPROVED** [review]
Didn't review much, looks OK to me.

File: examples/fullstack/backend/README.md [hunk 1]
```diff
@@ -77,7 +77,8 @@ Listening on :8080
 From another terminal, send a JSON request:
 
 ```shell
-$ curl -X POST -H "content-type: application/json" -d '{ "name": "World" }' -i -w "\n" localhost:8080
+$ curl -X POST -H "content-type: application/json" \
```
Comment (grant) [inline examples/fullstack/backend/README.md:80]: Can we remove the "$" so this snippet is copy-pastable?

File: examples/fullstack/frontend-cli/bin/greet.dart [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import 'dart:async';
```
Comment (kevmoo) [inline examples/fullstack/frontend-cli/bin/greet.dart:1]: copyright needed?

**Review by grant — APPROVED** [review]
1 nit

**Review by kevmoo — APPROVED** [review]

File: examples/fullstack/backend/README.md (hunk 1 above)
Comment (subfuzion) [reply examples/fullstack/backend/README.md]: I can, but what we really need to do is update the templates that generate these projects, which are based on our other examples.... Should probably open an issue for that. But I'll fix this for now.

File: examples/fullstack/frontend-cli/bin/greet.dart (hunk 2 above)
Comment (subfuzion) [reply examples/fullstack/frontend-cli/bin/greet.dart:15]: 👍

**Pull request merged**